### ISSUE TITLE
Can't Parse "mile" or "mi" to USCustomary.MILE

### ISSUE THREAD

**Issue by Weizilla** [unitsofmeasurement/uom-systems#50]
I'm trying to parse the string `mile` into `USCustomary.MILE` but I get a parse exception instead.

```java
UnitFormat unitFormat = SimpleUnitFormat.getInstance();
Unit<Length> mileUnit = unitFormat.parse("mile").asType(Length.class);
System.out.println(mileUnit);
```
throws 
```java
javax.measure.format.ParserException: Parse Error
	at tec.uom.se.format.SimpleUnitFormat$DefaultFormat.check(SimpleUnitFormat.java:561)
	at tec.uom.se.format.SimpleUnitFormat$DefaultFormat.parseSingleUnit(SimpleUnitFormat.java:414)
	at tec.uom.se.format.SimpleUnitFormat$DefaultFormat.parseProductUnit(SimpleUnitFormat.java:425)
	at tec.uom.se.format.SimpleUnitFormat.parseObject(SimpleUnitFormat.java:269)
	at tec.uom.se.format.SimpleUnitFormat$DefaultFormat.parse(SimpleUnitFormat.java:765)
	at tec.uom.se.format.SimpleUnitFormat$DefaultFormat.parse(SimpleUnitFormat.java:760)
	at tec.uom.se.format.SimpleUnitFormat$DefaultFormat.parse(SimpleUnitFormat.java:751)
```

I tried parsing `mi` as well but I get the same error.

Is unit parsing not supported for miles? It works with meters. Or perhaps I'm using the wrong set of libraries?

This is my list of maven dependencies:

```xml
  <dependencies>
        <dependency>
            <groupId>javax.measure</groupId>
            <artifactId>unit-api</artifactId>
            <version>1.0</version>
        </dependency>
        <dependency>
            <groupId>tec.uom</groupId>
            <artifactId>uom-se</artifactId>
            <version>1.0.2</version>
        </dependency>
        <dependency>
            <groupId>tec.uom.lib</groupId>
            <artifactId>uom-lib-common</artifactId>
            <version>1.0.1</version>
        </dependency>
        <dependency>
            <groupId>systems.uom</groupId>
            <artifactId>systems-common</artifactId>
            <version>0.5</version>
        </dependency>
        <dependency>
            <groupId>systems.uom</groupId>
            <artifactId>systems-quantity</artifactId>
            <version>0.5</version>
        </dependency>
        <dependency>
            <groupId>systems.uom</groupId>
            <artifactId>systems-common-java8</artifactId>
            <version>0.5</version>
        </dependency>
    </dependencies
```

Could it be related to https://github.com/unitsofmeasurement/uom-systems/issues/39? When I print `USCustomary.MILE`, I get `m*201168.0/125.0` instead of `mile` so could it be I'm missing some resource?

**Comment by keilw** [comment]
It could be. Would it be possible to try 0.6-SNAPSHOT of systems-common-java8? You find the additional repositories for the snapshot builds in every parent POM like https://github.com/unitsofmeasurement/uom-systems/blob/master/pom.xml
Will have a look and test it after getting back to Germany. It may be fixed in the Snapshot version together with UOM-SE 1.0.2 already.

**Comment by Weizilla** [comment]
Using 0.6-SNAPSHOT prints `mi` and parses `mi` successfully but fails on parsing `mile`.  Is there a way to parse `mile` too?

**Comment by keilw** [comment]
Yes, it's not an API element right now, but with alias() you can do, see
`	    SimpleUnitFormat.getInstance().alias(USCustomary.MILE, "mile");
	    
	    Unit u = SimpleUnitFormat.getInstance().parse("mile");`

**Comment by keilw** [comment]
See https://github.com/unitsofmeasurement/uom-demos/blob/master/console/systems/common/src/main/java/tec/uom/demo/systems/common/CommonFormatDemo.java and its Java SE 8 equivalent.

**Comment by keilw** [comment]
Addressed in 0.6, will close. Plan to release 0.6 after a few more tests.
